util/stmtsummary/statement_summary_test.go

```diff
@@ -48,30 +42,27 @@ func fakePlanDigestGenerator() string {
 	return "point_get"
 }
 
-func (s *testStmtSummarySuite) SetUpSuite(c *C) {
-	s.ssMap = newStmtSummaryByDigestMap()
-	err := s.ssMap.SetEnabled("1", false)
-	c.Assert(err, IsNil)
-	err = s.ssMap.SetRefreshInterval("1800", false)
-	c.Assert(err, IsNil)
-	err = s.ssMap.SetHistorySize("24", false)
-	c.Assert(err, IsNil)
-}
-
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	TestingT(t)
+func TestSetUp(t *testing.T) {
```

Comment:
I got a question about this test, what is it used for? Seems that it is not setup anything? Is it used for test function like `newStmtSummaryByDigestMap`

Reply:
ok, thinks for your explanation